pkg/universe.dagger.io/go/build.cue

```diff
@@ -38,8 +41,22 @@ import (
 			}
 		}
 		command: {
+			_packages: [...string] | []
+			if package == null && len(packages) == 0 {
+				_packages: ["."]
+			}
+			if package != null && len(packages) == 0 {
+				_packages: [package]
+			}
+			if package == null && len(packages) > 0 {
+				_packages: packages
+			}
+			if package != null && len(packages) > 0 {
+				_packages: [package] + packages
+			}
+
```

Comment:
Add a `FIXME: find a better workaround with disjunction`

Maybe someone will find it :cry:

Reply:
Also, just if we could factor the one in the build.cue and test.cue, that would be good. I didn't succeed, though.